kpops/component_handlers/kafka_connect/model.py

```diff
@@ -46,6 +47,23 @@ class KafkaConnectConfigErrorResponse(BaseModel):
     configs: list[KafkaConnectConfigDescription]
 
 
+class KafkaConnectResetterHelmConfig(BaseModel):
+    helm_config: HelmRepoConfig = Field(
+        default=HelmRepoConfig(
+            repository_name="bakdata-kafka-connect-resetter",
+            url="https://bakdata.github.io/kafka-connect-resetter/",
+        ),
+        description="Configuration of Kafka connect resetter Helm Chart",
+    )
+    version: str = "1.0.4"
+    helm_values: dict = Field(
+        default_factory=dict,
+        description="Overriding Kafka Connect Resetter Helm values. E.g. to override the Image Tag etc.",
+    )
+    namespace: str = Field(default="")
```

Comment:
should we make this a required argument? or what is the reason for `default=""`?